packages/esm-patient-list-app/src/api/hooks.ts

```diff
@@ -144,3 +141,19 @@ export function useCohortTypes() {
   const swrResult = useSWR<FetchResponse<CohortResponse<CohortType>>, Error>(`${cohortUrl}/cohorttype`, openmrsFetch);
   return { ...swrResult, data: swrResult?.data?.data?.results };
 }
+
+export function useCurrentUser() {
```

Comment:
User properties are returned as part of the session, so this hook seems redundant...